cisco_aci/datadog_checks/cisco_aci/fabric.py

```diff
@@ -209,3 +243,71 @@ def get_fabric_type(self, obj_type):
             return 'pod'
         if obj_type == 'l1PhysIf':
             return 'port'
+
+    def batch_payloads(self, devices, interfaces, collect_ts):
+        for device in devices:
+            yield NetworkDevicesMetadata(namespace=self.namespace, devices=[device], collect_timestamp=collect_ts)
+
+        payloads = []
+        for interface in interfaces:
+            if len(payloads) == PAYLOAD_METADATA_BATCH_SIZE:
+                yield NetworkDevicesMetadata(
+                    namespace=self.namespace, interfaces=payloads, collect_timestamp=collect_ts
+                )
+                payloads = []
+            payloads.append(interface)
+        if payloads:
+            yield NetworkDevicesMetadata(namespace=self.namespace, interfaces=payloads, collect_timestamp=collect_ts)
+
+    def submit_node_metadata(self, node_attrs, tags):
+        node = Node(attributes=node_attrs)
+        id_tags = ['namespace:{}'.format(self.namespace), 'system_ip:{}'.format(node.attributes.address)]
```

Comment:
```suggestion
        id_tags = ['device_namespace:{}'.format(self.namespace), 'device_ip:{}'.format(node.attributes.address)]
```
Do we add the `device_namespace` and `device_ip` tags to all Cisco ACI metrics?

Reply:
i have a separate [ticket](https://datadoghq.atlassian.net/browse/NDMII-2852) for that work ! i'm doing that in the bg, but if you prefer it all in one PR i can amend that!